src/capi.ts

```diff
@@ -109,6 +109,7 @@ const capiEndpoint = (articleId: string, key: string): string => {
       'show-fields': 'all',
       'show-tags': 'all',
       'show-blocks': 'all',
+      'show-elements': 'all',
```

Comment:
Do we need all or just one field?

Reply:
Sadly `elements` is [an array](https://github.com/guardian/apps-rendering/pull/189/files#diff-ebb6d8b623f26b27144c95b4ab728dddR305) 😔